### ISSUE TITLE
Error during fresh install, upon indexing database

### ISSUE THREAD

**Issue by ellingen** [humhub/humhub#2978]
### What steps will reproduce the problem?

Running the web installer results in an error after configuring the database.

### What is the expected result?

Database setup to complete.

### What do you get instead?

```
Database Exception – yii\db\Exception
SQLSTATE[HY000]: General error: 1709 Index column size too large. The maximum column size is 767 bytes.
The SQL being executed was: ALTER TABLE `user_module` ADD UNIQUE INDEX `index_user_module` (`user_id`, `module_id`)
Error Info: Array
(
    [0] => HY000
    [1] => 1709
    [2] => Index column size too large. The maximum column size is 767 bytes.
)

↵
Caused by: PDOException
SQLSTATE[HY000]: General error: 1709 Index column size too large. The maximum column size is 767 bytes.
in /var/www/virtual/linden90/html/protected/vendor/yiisoft/yii2/db/Command.php at line 842
```

### Additional info

| Q                | A
| ---------------- | ---
| HumHub version   | 1.2.4
| MariaDB | 10.1.30-MariaDB MariaDB Server
| PHP version      |  7.1.13
| Operating system | CentOS 7

> Note: Please use the [community](https://community.humhub.com) site for any questions on customizing HumHub.
> Note: Also provide the full error stacks from `Administration --> Information --> Logging` and errors from your browser console if possible.

**Comment by buddh4** [comment]
Which Character Set did you use for your database? I think the current master does not support `utf8mb4` this should be fixed in the 1.3-dev repository.

**Comment by ellingen** [comment]
@buddh4 Thanks! Changing the character set to `utf8_general_ci` solved it.
